### PR TITLE
WIP: Add first draft of README

### LINKED ISSUE #2: Add a README
Please, add a README. We are not savages.

### PR DESCRIPTION
**Pull request by frenzymadness**
Fix #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# python-ethtool
+
+*Python bindings for the ethtool kernel interface*
+
+**This is a new upstream for python-ethtool maintained by Red Hat/Fedora
```
Comment (stratakis) [inline README.md]: As this is the fedora-python github organization I would suggest to remove the Red hat/ Fedora Python maintainance team line. Maybe just put Fedora's Python SIG?

Comment (frenzymadness) [inline README.md]: Ok, thanks. Fixed

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Python SIG.**
+
+## About
+
+Allows querying and changing of ethernet card settings, such as speed,
+port, autonegotiation, and PCI locations.
+
+## Authors
+
+### Original authors
```
Comment (hroncok) [inline README.md]: I'd keep that unified

**Comment by hroncok** [comment]
Some things missing:

 * [ ] License information
 * [ ] Example usage
 * [ ] Mention manpage
 * [ ] How to run tests (keep that omitted for now, as it will change with https://github.com/fedora-python/python-ethtool/issues/3)

**Comment by hroncok** [comment]
Especially with project, that has a history with moving upstream, plese always reference issues and pull request in the commit messages with full URLs:

```
Fix https://github.com/fedora-python/python-ethtool/pull/2
```

Instead of:

```
Fix #2
```

GitHub will show it nicely anyway, but if the repo is moved, the numbers keep making sense.

**Comment by hroncok** [comment]
Also, since this should end up in ;long_description` and on PyPI, could you please use RST instead of Markdown?

**Comment by hroncok** [comment]
Closing in favor of https://github.com/fedora-python/python-ethtool/pull/13

**Pull request closed without merging**